src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/validering/VedtaksperiodeValideringUtilsTest.kt

```diff
@@ -364,6 +364,22 @@ class VedtaksperiodeValideringUtilsTest {
                 assertThat(feil.feil).contains("Kan ikke innvilge når det ikke finnes utgifter hele vedtaksperioden")
             }
 
+            @Test
+            fun `kaster feil hvis det ikke finnes alle utgifter i hele vedtaksperioden`() {
```

Comment:
Litt rar formulering her 😉 
```suggestion
            fun `kaster feil hvis det ikke finnes utgifter i hele vedtaksperioden`() {
```